client/components/dashboard/dashboard-directive.html

```diff
@@ -10,8 +10,7 @@
                ng-model="container.model"
                ng-click="clickContainer($event, container)">
       <div ng-show="container.model.container.header_text"
-           class="pk-container-header">
-        {{ container.model.container.header_text }}
+           class="pk-container-header" btf-markdown="container.model.container.header_text">
```

Comment:
Can you add explanations somewhere about security / sanitization? It does seem to be happening, i.e. `<a href="javascript:alert(42)">test</a>` doesn't get linked, but for variations of this such as `<a href="javascript:alert('test')">test</a>` I get uncaught parse errors thrown on the JS console.